lua/nvim-tree/renderer/init.lua

```diff
@@ -111,6 +116,7 @@ function M.setup(opts)
   }
 
   _padding.setup(opts)
+  git.setup_signs()
```

Comment:
should we clear/setup signs on each draw ?

Reply:
It is working like this. Changing highlight groups immediately updates.